### PR TITLE
Add spastorino to mailmap

### PR DESCRIPTION
**Pull request by spastorino**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -495,6 +495,8 @@ Ryan Wiedemann <Ryan1729@gmail.com>
 S Pradeep Kumar <gohanpra@gmail.com>
 Sam Radhakrishnan <sk09idm@gmail.com>
 Samuel Tardieu <sam@rfc1149.net>
+Santiago Pastorino <spastorino@gmail.com>
+Santiago Pastorino <spastorino@gmail.com> Santiago Pastorino <santiago@wyeworks.com>
```
Comment (compiler-errors) [inline .mailmap:499]:
```suggestion
Santiago Pastorino <spastorino@gmail.com> <santiago@wyeworks.com>
```

I think this works?

Comment (spastorino) [reply .mailmap:499]: I have no idea :), I think so but the .mailmap file has tons of entries with the name repeated. Anyway going to change it.

**Comment by compiler-errors** [comment]
i don't think that first line is necessary but whatever

**Pull request merged**